jupyterhub/app.py

```diff
@@ -365,12 +366,33 @@ def _log_format_default(self):
         """override default log format to include time"""
         return "%(color)s[%(levelname)1.1s %(asctime)s.%(msecs).03d %(name)s %(module)s:%(lineno)d]%(end_color)s %(message)s"
 
+    extra_log_file = Unicode("", config=True)
+    extra_log_handler = Instance(
+        klass=logging.Handler,
+        allow_none=True,
+        config=True,
```

Comment:
I could imagine wanting to set, e.g., a `RotatingFileHandler` or `SysLogHandler`.  I figured the common case of "log to a file instead of stderr" was useful enough to support from the command line, and if people want a more exotic setup they can do it via config.